src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,63 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static MovieService movieService
+            = (MovieService) injector.getInstance(MovieService.class);
+    private static CinemaHallService cinemaHallService
+            = (CinemaHallService) injector.getInstance(CinemaHallService.class);
+    private static MovieSessionService movieSessionService =
+            (MovieSessionService) injector.getInstance(MovieSessionService.class);
+
     public static void main(String[] args) {
-        MovieService movieService = null;
+        //        movieService.getAll().forEach(System.out::println);
+        //        cinemaHallService.getAll().forEach(System.out::println);
+        movieSessionService.findAvailableSessions(1L, LocalDate.now()).forEach(System.out::println);
+    }
 
+    public static void injectMovie() {
         Movie fastAndFurious = new Movie("Fast and Furious");
-        fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
+        fastAndFurious.setDescription(
+                "An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
-        movieService.getAll().forEach(System.out::println);
+
+        Movie terminator = new Movie("Terminator 5");
+        fastAndFurious.setDescription("hasta la vista baby");
+        movieService.add(terminator);
+    }
+
+    public static void injectCinemaHall() {
+        CinemaHall lavina = new CinemaHall(500, "Lavina Mall");
+        CinemaHall multiplex = new CinemaHall(2000, "multiplex");
+        cinemaHallService.add(lavina);
+        cinemaHallService.add(multiplex);
+    }
+
+    public static void injectMovieSessions() {
```

Comment:
why you dont use `injectMovieSessions()` `injectCinemaHall()` `injectMovie()` method anywhere?